### ISSUE TITLE
PACKER_CACHE_DIR not working with remove ESXI VMWare builder

### ISSUE THREAD

**Issue by timc3** [mitchellh/packer#1218]
I set PACKER_CACHE_DIR env variable thinking it would just effect the local files paths used and not those used on the remove ESXi machine ( a mistake if you ask me, they should be separate configuration options ).

My VMWare Fusion and Virtual Box builders continued to work, but the remote ESXi stopped after uploading to the remove PACKER_CACHE_DIR:

**Comment by icnocop** [comment]
Hi.

I believe I ran into the same issue.

Using packer 0.7.5 windows amd64.

I'm trying to build a Windows virtual machine on a VMware ESXi 5.5 server.

The ISO already exists in "/vmfs/volumes/MyDatastore1/ISOs".

I haven't set PACKER_CACHE_DIR, but I've set remote_cache_*.

Here is packer.json:

```
{
    "builders": [
        {
            "disk_size": 40960,
            "floppy_files": [
                "floppy/autounattend.xml",
                "floppy/install-winrm.cmd"
            ],
            "guest_os_type": "windows8srv-64",
            "iso_url": "en_windows_server_2012_r2_x64_dvd_2707946.iso",
            "iso_checksum_type": "sha1",
            "iso_checksum": "B6F063436056510357CB19CB77DB781ED9C11DF3",

            "type": "vmware-iso",
            "remote_type": "esx5",
            "remote_host": "MyRemoteHost",
            "remote_datastore": "MyDatastore",
            "remote_username": "root",
            "remote_password": "password",
            "remote_cache_datastore": "/vmfs/volumes/MyDatastore1",
            "remote_cache_directory":"ISOs",
            "ssh_username": "vagrant",
            "shutdown_command": "shutdown -a; shutdown /s /t 1 /c \"Packer Shutdown\" /f /d p:4:1",
            "vmx_data": {
                "gui.fitguestusingnativedisplayresolution": "FALSE",
                "memsize": "4096",
                "numvcpus": "1",
                "virtualHW.version": "10",
                "scsi0.virtualDev": "lsisas1068"
            }
        }
    ]
}
```

Here is the output from running 'packer.exe build packer.json':
```
vmware-iso output will be in this color.

==> vmware-iso: Downloading or copying ISO
    vmware-iso: Downloading or copying: file:///vmfs/volumes/MyDatasto
re1/ISOs/en_windows_server_2012_r2_x64_dvd_2707946.iso
    vmware-iso: Error downloading: open vmfs/volumes/MyDatastore1/ISOs
/en_windows_server_2012_r2_x64_dvd_2707946.iso: The system cannot find the path
specified.
==> vmware-iso: ISO download failed.
Build 'vmware-iso' errored: ISO download failed.

==> Some builds didn't complete successfully and had errors:
--> vmware-iso: ISO download failed.

==> Builds finished but no artifacts were created.
```

Any ideas/advice? (This is my first attempt with packer)

Thank you.

**Comment by michaeltlombardi** [comment]
Should this issue have been closed by #1479?

**Comment by rickard-von-essen** [comment]
Closed by #1479
